hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/graphite/GraphiteClient.java

```diff
@@ -32,11 +32,10 @@ public GraphiteClient(WebTarget webTarget) {
     public GraphiteMetrics readMetrics(String... metrics) {
         try {
             GraphiteMetrics response = new GraphiteMetrics();
-            queryGraphite(metrics).stream()
-                    .forEach(metric -> response.addMetricValue(metric.getTarget(), getFirstValue(metric)));
+            queryGraphite(metrics).stream().forEach(metric -> response.addMetricValue(metric.getTarget(), getFirstValue(metric)));
             return response;
         } catch (GraphiteConnectionException exception) {
-            logger.warn("Unable to read from Graphite", exception);
+            logger.warn("Unable to read from Graphite", exception.getMessage());
```

Comment:
Why do we remove stacktrace information from our logs? GraphiteConnectionException messages are not very informative, e.g. "Error connecting to Graphite". Now we won't know the cause: timeout, corrupted data, host unreachable or something else?